<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMessageDialog</class>
 <widget class="QDialog" name="LoginMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="LoginMessageLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
